RELEASE.md

```diff
@@ -24,6 +24,22 @@ In the below example, we will pretend that we're upgrading from `0.11.0` to `0.1
 1. Edit the new release https://github.com/containers/podman-desktop/releases/edit/v0.12.0
 1. Select previous tag (v0.11.0) and click on `Generate release notes` and the click on `Update release`
 
+## Test release before it is rolling out.
+
+The release is a pre-release, it means it is not yet the latest version, so no clients will automatically update to this version.
+
+It allows to make QE tests of these binaries before they go live or are in the package managers.
```

Comment:
Adding some suggestions based on previous comment, feel free to not take literally or reword if you have a better idea.

`It allows to make QE tests of` =>
`It allows QE (and everyone else) to test`?